package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "start": "node .",
     "stop": "pkill levweb",
-    "dev": "NODE_ENV=development nodemon .",
+    "dev": "npm run mock-kc-proxy & NODE_ENV=development nodemon .",
```

Comment:
OR, maybe we should have a special dev.js that acts as the entry point instead of app.js. Then we can make it work the same way as in the acceptance tests.